README.md

```diff
@@ -129,6 +129,12 @@ This command will retrieve the given package manager from the specified archive
 
 - `COREPACK_ROOT` has no functional impact on Corepack itself; it's automatically being set in your environment by Corepack when it shells out to the underlying package managers, so that they can feature-detect its presence (useful for commands like `yarn init`).
 
+- `COREPACK_NPM_REGISTRY` sets the registry base url used when retrieving package managers from npm. Default value is `https://registry.npmjs.org`
+
+- `COREPACK_NPM_TOKEN` sets a Bearer token authorization header when connecting to `npm` type registry.
+
+- `COREPACK_NPM_USERNAME` and `COREPACK_NPM_PASSWORD` to set a Basic authorization header when connecting to NPM type registry. Note: both environment variables are required, as plain text. If you want to send an empty password, explicitly set `COREPACK_NPM_PASSWORD` to an empty string.
```

Comment:
```suggestion
- `COREPACK_NPM_USERNAME` and `COREPACK_NPM_PASSWORD` to set a Basic authorization header when connecting to npm type registry. Note: both environment variables are required, as plain text. If you want to send an empty password, explicitly set `COREPACK_NPM_PASSWORD` to an empty string.
```
https://github.com/npm/cli/tree/37bf6d31bd2f68e661928744833b57dcabbb0d1a#is-it-npm-or-npm-or-npm

Reply:
Today I learned! Thank you for the insight as to npm capitalisation